include/zephyr/bluetooth/audio/aics.h

```diff
@@ -364,6 +373,24 @@ int bt_aics_unmute(struct bt_aics *inst);
  */
 int bt_aics_mute(struct bt_aics *inst);
 
+/**
+ * @brief Set manual only gain mode in Audio Input Control Service.
+ *
+ * @param inst          The instance pointer.
+ *
+ * @return 0 on success, errno value on fail.
+ */
+int bt_aics_gain_set_manual_only(struct bt_aics *inst);
+
+/**
+ * @brief Set automatic only gain mode in Audio Input Control Service.
+ *
+ * @param inst          The instance pointer.
+ *
+ * @return 0 on success, errno value on fail.
+ */
```

Comment:
Maybe we should add some documentation about this disabling support for setting the gain via `bt_aics_gain_set` or from clients